recipes/ipyrad/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ipyrad" %}
+{% set version = "0.9.7" %}
+{% set sha256 = "ee1afb26eff0ee779e12e968f3d1ff8415105112046f0e192dff8cdd85190c88" %}
+
+package:
+  name: {{name}}
+  version: {{version}}
+
+source:
+  url: https://github.com/eaton-lab/{{name}}/archive/{{version}}.tar.gz
+  sha256: {{sha256}}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: generic
```

Comment:
@dpryan79 Why `generic` and not `python`? Does this install things outside of `site-libraries`?

Reply:
Oh, I missed that!

Reply:
Hey all, I did try with `noarch: python` but was having problems with build numbers. Sorry for the confusion, I didn't realize I could re-open this pull request after it was closed so I started a new one: #15791

Reply:
You can re-open merged PRs??? Ugh... really?